core/src/main/scala/com/spotify/tfexample/derive/TensorflowMapping.scala

```diff
@@ -79,6 +79,10 @@ object TensorflowMapping {
   def toStrings(f: Feature): Seq[String] = toByteStrings(f).map(_.toStringUtf8)
 
   def fromByteBuffers(xs: Seq[ByteBuffer]): Feature.Builder =
-    fromByteArrays(xs.map(_.array()))
+    fromByteStrings(xs.map { buf =>
+      val byteStr = ByteString.copyFrom(buf)
+      buf.rewind()
```

Comment:
Hm, I'm wondering why the rewind is necessary here. We're not using `buf` again, right? And it's not about preserving the original mark (which could have been non-zero).

If anything, shouldn't we be rewinding before the `copyFrom`?

Reply:
That's a good point. The purpose of this is to leave the buffer in the same state it was in before making the copy, but it's true that it can start with a non-zero position. I can fix that to rewind by the same number of bytes that were read. Make sense?